### DISCUSSION TITLE
[QUESTION]:用户手册提供的dockers使用教程执行过程中发现输出的日志文件找不到

### DISCUSSION THREAD

**Discussion by lpq131004** [alibaba/ilogtail#1343] [Help] [answered]
按照[教程](https://ilogtail.gitbook.io/ilogtail-docs/installation/start-with-container)
启动logtail容器后，查看采集到的标准输出`cat /usr/local/ilogtail/simple.stdout`
报错：No such file or directory

**Comment by messixukejia** [comment, marked as answer]
1、可以进入容器看看ilogtail的运行状态
2、看下/usr/local/ilogtail/是否挂载正常
3、docker logs docker_ilogtail是否执行构造了标准输出？
